### PR TITLE
Proprietary gpu 2

### PR DESCRIPTION
**Pull request by camckay**
Filled out documentation to included full steps via YAST and CLI. Added steps for hybrid graphics.

### CODE REVIEW

File: project/docs/install_proprietary.md [hunk 1]
```diff
@@ -2,27 +2,52 @@
...
+2. The __nvidia-glG05__ package corresponds with the Nvidia 460 series driver.
+3. The __nvidia-glG04__ package corresponds with the Nvidia 390 series driver.
 4. Please refer to the Nvidia [website](https://www.nvidia.com/en-us/drivers/unix/) to determine which driver best supports your GPU.
 
 ### Setup the driver
 
+####With Yast
 1. Go to _YAsT2_.
 2. Then _Software Management_.
 3. On the menu, click __Configuration__ > __Repositories__... (or do `Ctrl + R`).
```
Comment (daviddyess) [inline project/docs/install_proprietary.md:15]: Please use `&gt;` instead of `>`

**Review by daviddyess — CHANGES_REQUESTED** [review]

File: project/docs/install_proprietary.md (hunk 1 above)
Comment (camckay) [reply project/docs/install_proprietary.md]: Done.

**Review by Nycticorax — CHANGES_REQUESTED** [review]
Thanks for this PR, great addition to the page! Just a quick comment:

"Recent AMD APUs and Nvidia" is just like "OFFLOADING SPECIFIC APPLICATIONS TO THE NVIDIA GPU" except for the line reading "The open source AMD GPU drivers will load by default.". So I wonder if "Recent AMD APUs..." requires a separate sections of its own. If the point you are making is that setting up drivers for new APUs is very similar to what you've described already, what about removing the section and adding instead a note or info box like
```
??? info "Recent AMD APUs and Nvidia"
    The above instructions work the same (your words here).
```
? Similar examples can be found under `desktop_choice.md`.

**Comment by camckay** [comment]
> f the point you are making is that setting up drivers for new APUs is very similar to what you've described already

I see your point. My thoughts in creating a separate section were twofold...

1. If someone is scanning the page looking for their particular hardware combination, the information needed would mostly be in that section.
2. This is a relatively new combination in laptops. So I only have a sample size of 1 - I have a single laptop with this configuration. So I wasn't sure if my experience would hold, or if others might have something to add to this section.

**Comment by Nycticorax** [comment]
> > f the point you are making is that setting up drivers for new APUs is very similar to what you've described already
> 
> I see your point. My thoughts in creating a separate section were twofold...
> 
> 1. If someone is scanning the page looking for their particular hardware combination, the information needed would mostly be in that section.
> 2. This is a relatively new combination in laptops. So I only have a sample size of 1 - I have a single laptop with this configuration. So I wasn't sure if my experience would hold, or if others might have something to add to this section.

Okay, let's go that road then. A few quirks:

- If we follow the logic your suggesting, we should fix the names in the entire document. Can you do it? "Nvidia driver" => "GPU drivers" or something along these lines.
- If we follow this logic we should also change the structure. Can you do it? I have in mind somethng like i.e.
  - Nvidia dedicated + Intel integrated
  - Nvidia dedicated
  - AMD dedicated & integrated (is the same as AMD APUs?)
  - AMD dedicated
- "Recent AMD APUs and Nvidia" had a redundant numbering. Fixed it myself.

**Comment by Nycticorax** [comment]
> > f the point you are making is that setting up drivers for new APUs is very similar to what you've described already
> 
> I see your point. My thoughts in creating a separate section were twofold...
> 
> 1. If someone is scanning the page looking for their particular hardware combination, the information needed would mostly be in that section.
> 2. This is a relatively new combination in laptops. So I only have a sample size of 1 - I have a single laptop with this configuration. So I wasn't sure if my experience would hold, or if others might have something to add to this section.

Okay, so let's follow this logic to the end: can you restructure -- and rename when required -- the sections to cover these use cases? One reasonable structure seems to go like:
  - Intel integrated + Nvidia dedicated
  - Nvidia dedicated
  - Intel integrated + AMD dedicated
  - AMD dedicated
 I don't know where AMD APUs would fit in this structure, but I am sure you'll find a sweet spot :)

**Comment by Nycticorax** [comment]
@camckay still around?

**Comment by vegetable-broth** [comment]
I notice we should use https://github.com/openSUSE/SUSEPrime and say more on how to power off the NVidia GPU when not in use.

**Comment by camckay** [comment]
I agree. Should I split that section out then and push it to a new section?

On Jan 28, 2021 11:06 AM, Ad <notifications@github.com> wrote:

I notice we should use https://github.com/openSUSE/SUSEPrime<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FopenSUSE%2FSUSEPrime&data=04%7C01%7C%7C1c55a16bc4cc433084c808d8c3af079d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637474503797981927%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=7CjjLBrYmoxBWQfAmb9K7hFB0wdbMMXNemD2aCn3%2F5k%3D&reserved=0> and say more on how to power off the NVidia GPU when not in use.

**Comment by vegetable-broth** [comment]
> I agree. Should I split that section out then and push it to a new section?

Would be great!

**Comment by camckay** [comment]
Okay, I think the way I have it set up now looks quite a bit better. I got separate doc created for the hybrid graphic config, and tried to add some more clarity to that section, with a clearer distinction between onboard AMD and Intel graphics. But I combined the section on offloading, since it is the same regardless of which onboard gpu you use.

**Review by why-not-try-calmer — APPROVED** [review]
very cool, it's indeed better! let's continue this conversation in the Issues.

**Pull request merged**